pkg/csi_driver/utils.go

```diff
@@ -59,3 +62,13 @@ func logGRPC(ctx context.Context, req interface{}, info *grpc.UnaryServerInfo, h
 	}
 	return resp, err
 }
+
+// IsNotCIDR verifies if reserved IP range is specified,
+// it must be a named address range instead of direct CIDR value.
+func IsNotCIDR(ipRange string) bool {
+	_, _, err := net.ParseCIDR(ipRange)
+	if err == nil {
+		return false
+	}
+	return strings.Contains(err.Error(), "invalid CIDR address")
+}
```

Comment:
actually here how this function is called is probably not concise - here the function reverses the "is something CIDR" and then when it was used the result is !-ed again.

So my suggestion is actually just to write a "IsCIDR" and make the whole thing simpler.

In fact, I believe you can directly treat `if _, _, err := net.ParseCIDR(ipRange); err == nil` as the condition that a string `ipRange` is a CIDR range and false otherwise without needing to match "invalid CIDR address" in the error it returned.

Reply:
Agree, revised function to IsCIDR.